src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.cpp

```diff
@@ -48,7 +48,7 @@ using HandleAndValueVector = std::vector<std::pair<AttribHandle<T>, T>,
                                          Eigen::aligned_allocator<std::pair<AttribHandle<T>, T>>>;
 
 template <typename T>
-void copyAttribToCoreVertex( HandleAndValueVector<T>& data, TopologicalMesh* topoMesh,
+void copyAttribToCoreVertex( HandleAndValueVector<T>& data, const TopologicalMesh* topoMesh,
                              std::vector<PropPair<T>>& vprop,
```

Comment:
why not `const ...& vprop` ?

Reply:
same at line 61, why not `const ...& data` ?

Reply:
Actually, I'm not aware of what copyAttrib as to do, I just propagate constness from the toTriangleMesh operator.
So feel free to add const if appropriate.